docassemble/ALDashboard/data/questions/install_assembly_line.yml

```
@@ -198,31 +218,111 @@ fields:
     hide if: skip_efile
     required: False
 ---
-id: feedback form config
+# This must be tested with a private repository
+#  If your GitHub organization's repositories are private, the account must have permission to see it and file issues.
+id: github PAT
 continue button field: feedback_form_config
 question: |
-  GitHub feedback and about information
+  Feedback and "About" page information with GitHub
 subquestion: |
-  You should configure a dedicated GitHub account with permission to make
-  pull requests on public repositories and generate a private access token
-  for it.
+  Assembly Line uses GitHub to let you:
+  
+  1. Get user feedback about each interview.
+  2. Display a "last updated" date on the "About" page of an interview.
+
+  First, in order to use GitHub you should create a dedicated GitHub account. You can use an account that already exists, but we recommend having an account that only handles Assembly Line functionality.
+  
+  The account must have permission make pull requests on your GitHub organization's repositories. You can read [GitHub's documentation about setting an account's permissions](https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization).
   
-  This feature is used to add feedback to production interviews as well
-  as to display a "last updated" date on the about page.
+  Give us the information about that account and set up its permissions.
 fields:
+  - What is the username of the account: the_config['github issues']['username']
+    default: ${ the_config['github issues'].get('username', '') }
+    help: |
+      This will set the `config['github issues']['username']` and the `config['github readonly']['username']` values.
   - note: |
-      #### GitHub Feedback form
-  - Username: the_config['github issues']['username']
-  - Private token: the_config['github issues']['token']
-  - Default Github organization: the_config['github issues']['default repository owner']
-  - Allowed repository owners (one per line): allowed_github_repo_owners
+      Now you need to authorize your docassemble server to use that account to make issues. You can do that by creating a personal access token (PAT):
+
+      1. Follow directions to start making a personal access token (PAT) for the account on GitHub. You can find the directions in a section at [this GitHub documentation page](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
+      1. Set the expiration date to "No expiration".
+      1. Tap the checkbox for “repo” permissions.
```

```suggestion
      1. Follow [these directions to make a personal access token (PAT) for the account](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) on GitHub.
          1. Set the expiration date to "No expiration".
          1. Tap the checkbox for "repo" permissions.
```
Not sure if the indentation will work, but would be nice to show that these steps are when you are following those instructions. I removed the smart quotes here too.

---

There are reasons for some of that text.

1. I specifically said "start making" because I don't want them to complete the whole process without looking at further instructions.
2. The link doesn't lead directly to the correct section. The link leads to a bunch of other stuff too and there's no way to link to that section directly.

This might not be a better way to describe it though. One of the users we tested with found the less precise wording confusing, but we didn't get to test this new wording with anyone. I dunno.

As long as you take that into consideration, I'll do what I'm advised! The indentation does need to be tested, though.

---

Good point about direct linking, though you can link directly to the instructions: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token. That's a complete list of things to do, there shouldn't be too much other stuff there.

That's also fair about the less precise wording. Maybe something like this would be better then:
```suggestion
      1. Follow [these directions to make a personal access token (PAT) for the account](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) on GitHub, stopping at the "expiration" section.
      1. Set the expiration date to "No expiration".
      1. Tap the checkbox for "repo" permissions.
```